src/net.cpp

```diff
@@ -110,9 +111,9 @@ void CConnman::AddOneShot(const std::string& strDest)
     vOneShots.push_back(strDest);
 }
 
-unsigned short GetListenPort()
+uint16_t GetListenPort()
 {
-    return (unsigned short)(gArgs.GetArg("-port", Params().GetDefaultPort()));
+    return (uint16_t)(gArgs.GetArg("-port", Params().GetDefaultPort()));
```

Comment:
Can we just drop the cast entirely? Not sure I see the point...

Reply:
The point is to make a potentially sign-changing narrowing conversion explicit.

`GetArg` returns `int64_t` so it makes sense to make the implicit conversion to `uint16_t` explicit using a cast.